### DISCUSSION TITLE
[Community Call] Agenda – June 28, 2022

### DISCUSSION THREAD

**Discussion by natemoo-re** [withastro/roadmap#216] [Announcements]
## To Discuss

- **Announcement** Welcome `@team-dev`!
- **Announcement** Switching up the format
  - [`p2-to-be-discussed`](https://github.com/withastro/astro/issues?q=is%3Aopen+is%3Aissue+label%3Ap2-to-be-discussed) issues moving to a `@team-dev` call
  - More space for ecosystem initiatives, demos, etc
- **Announcement** It's been a busy week!
  - **54 PRs** and **22 Releases**
  > Excluding merges, **23 authors** have pushed **91 commits** to `main` and **234 commits** to all branches. On main, **371 files have changed** and there have been [6,566 additions and 3,146 deletions](https://github.com/withastro/astro/compare/e67171252c7df572b1ec72360ce0ec4a7f5a4883...main).
- **Announcement** @ran-dall and @aFuzzyBear working on OpenSSF compliance
  - https://github.com/withastro/astro/pull/3718
- **Help Wanted** Please give feedback on new RFCs!
  - [#210](https://github.com/withastro/rfcs/pull/210) **Astro.extend**
  - [#212](https://github.com/withastro/rfcs/pull/212) **client:visible update**
  - [#215](https://github.com/withastro/rfcs/pull/215) **Add style src="" attribute**
- **Help Wanted** New repo for [community health files](https://github.com/withastro/.github)
  - Readme on [`withastro`](https://github.com/withastro) could use some love
- **@natemoo-re** has some news! 
  - MDX support is coming in [`@astrojs/mdx`](https://github.com/withastro/astro/pull/3706)
- **@delucis** has some news!
  - `preact/compat` support is [in progress](https://github.com/withastro/astro/pull/3712)
- **@tony-sull** has some news!
  - Experimental `@astrojs/prefetch` just released
  - Experimental `@astrojs/image` [in progress](https://github.com/withastro/astro/pull/3694)
- **@sarah11918** has some docs updates!
  - **49 PRs** merged, **20 PRs** open
  - New [deploy](https://docs.astro.build/en/guides/deploy/) docs!

_If there is something that you would like to see discussed, please comment below. We will prioritize items already on the agenda, but if there is time to spare we will open up the call to get attention on and discuss [other threads and feature requests](https://github.com/withastro/rfcs/discussions)._

**Comment by delucis** [comment]
# Community Call Minutes, 28 June 2022

### News 📰

@natemoo-re reports what’s new in the Astroverse:

- 👥 New team on Discord — `@team-dev`. Go sign up!

- ☎️ New weekly issue triage call on Discord. Come find bugs with us 🔎🐛 

- 🖥️ Busy week with heaps of PRs merged and so many releases 🥳 

- 🔒 OpenSSF audit in progress: https://github.com/withastro/astro/pull/3718

- :octocat: New repo: https://github.com/withastro/.github for centralising Code of Conduct and other stuff. Help us make it cooler!

- 🏆 [The Astro Hackathon Showcase](https://hackathon-1-0-projects.vercel.app/) now highlights our amazing hackathon winners and honourable mentions. 

### Show and tell 🦔

- 🔜 @natemoo-re: MDX support via a new `@astrojs/mdx` integration [coming SOON](https://github.com/withastro/astro/pull/3706)!

- 🔜 @delucis: Support for rendering React components with `@astrojs/preact` using `preact/compat` — also [coming SOON](https://github.com/withastro/astro/pull/3712)!

- 🆕 New [`@astrojs/prefetch`](https://www.npmjs.com/package/@astrojs/prefetch) integration! 🎉 

   @tony-sull shows off the speed improvements this can bring to a site by prefetching links users might need.

   Try it today by running `astro add prefetch` to install and add `rel="prefetch"` to links you want to preload.

- 🔜 @tony-sull: New `@astrojs/image` integration! Will provide a higher-level API for different image providers to plug into. Follow along in the PR: https://github.com/withastro/astro/pull/3694

- ✅ Upgrade to Vite 3 is now passing CI! Planning to merge tomorrow.

   @bholmesdev: Speed improvements coming, will require some increased usage of Vite’s `ssr.noExternal` option.
   
   Follow along in the PR: https://github.com/withastro/astro/pull/3570

### Docs Report 📚

@sarah11918 updates us on the latest from docs:

- 🚀 Check out our new deploy pages: https://docs.astro.build/en/guides/deploy/

   **Do you use one of these services?**
   Help us by checking pages for accuracy and contributing with Issues or PRs to address anything you catch.

- 📝 New [contributing](https://github.com/withastro/docs/blob/main/CONTRIBUTING.md) and [writing](https://github.com/withastro/docs/blob/main/WRITING.md) guides.

   Read these if you have questions about how to get involved or are wondering how to get things written. Lots of different ways to join us. For example, PR reviews are super valuable contributions! Jump in! No PR merged without a second pair of eyes.

- @natemoo-re: Loving the recent design improvements
   @sarah11918: Docs is an Astro site — GREAT way to get experience working on a big Astro project in production.

### RFCs/Issues to be discussed 💬

Open for comments — to be discussed in next week’s call. Let us know your thoughts!

- #210

- #212

- 🆕 #215
